crm_tag_multi_company/security/crm_tag.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="crm_tag_comp_rule" model="ir.rule">
+        <field name="name">Crm Tag multi company rule</field>
+        <field name="model_id" ref="sales_team.model_crm_tag" />
+        <field name="domain_force">
+            ['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]
```

Comment:
Since https://github.com/odoo/odoo/commit/a04df5cb1411a14752ba7a4d6b3c4b7b4f1d6d44
these rules are written as follow in Odoo : 
```suggestion
            [('company_id', 'in', company_ids + [False])]
```

Maybe it should be better to follow this guideline ?